src/djcytoscape/templates/djcytoscape/quest_map.html

```diff
@@ -11,8 +11,12 @@
 {%endif %}
 {% endblock%}
 {% block content %}
+            <p>
+              <a class="btn btn-default" href="{% url 'djcytoscape:list' %}" role="button"
+              title = "List all maps" >
+              <i class="fa fa-list-ul"></i>
+            </a>
```

Comment:
Hmm... I think this looks weird.  Instead, can you put a button beside `{{ scape.name }} Quest Map` that says "All Maps" Similar to the buttons on many pages beside the headings.

Reply:
![image](https://user-images.githubusercontent.com/105619909/183758711-1b5bb4bf-d16e-4363-bad1-3dd369ef7f20.png)
map list button now displays next to map names.